apps/omg_watcher_info/lib/omg_watcher_info/db/block.ex

```diff
@@ -92,6 +92,56 @@ defmodule OMG.WatcherInfo.DB.Block do
     |> Paginator.set_data(paginator)
   end
 
+  @spec query_last_24_hour(Ecto.Query.t()) :: Ecto.Query.t()
+  def query_last_24_hour(query) do
+    now = DateTime.to_unix(DateTime.utc_now())
+    twenty_four_hours = 86_400
```

Comment:
This looks like a static truth. Can move to module annotation i.e.

```elixir
defmodule OMG.WatcherInfo.DB.Block do
  #...

  @seconds_in_twenty_four_hours 86_400

  #...
end

Reply:
Done in various files with 8946c62 and 6ea2c86.